dolfyn/tools/misc.py

```diff
@@ -275,11 +232,11 @@ def fillgaps(a, maxgap=np.inf, dim=0, extrapFlg=False):
         for i2 in range(0, inds.__len__()):
             ii = list(range(gd[inds[i2]] + 1, gd[inds[i2] + 1]))
             a[ii] = (np.diff(a[gd[[inds[i2], inds[i2] + 1]]]) *
-                     (np.arange(0, ii.__len__()) + 1) /
-                     (ii.__len__() + 1) + a[gd[inds[i2]]]).astype(a.dtype)
+                      (np.arange(0, ii.__len__()) + 1) /
+                      (ii.__len__() + 1) + a[gd[inds[i2]]]).astype(a.dtype)
 
 
-def interpgaps(a, t, maxgap=np.inf, dim=0, extrapFlg=False):
+def _interpgaps(a, t, maxgap=np.inf, dim=0, extrapFlg=False):
```

Comment:
I think we want to expose this function to the API. It's a widely used tool in ADCP analysis.